client/scripts/models/IChainAdapter.ts

```diff
@@ -63,6 +64,11 @@ abstract class IChainAdapter<C extends Coin, A extends Account<C>> {
       });
     }
 
+    if (this.meta.chain.id !== m.route.get().split('/')[1]) {
```

Comment:
this seems fragile, is there a way to use m.route.get() to get the routing parameter instead?

Reply:
Updated.